src/client/AM.PA.MacOs/PersonalAnalytics/Shared/SQLController.swift

```diff
@@ -64,18 +65,19 @@ class SQLController{
     
     func fetchAggregatedInputSince(time: Double) -> [AggregatedInputEntry] {
```

Comment:
This function should be renamed to fetchUserInputSince, same with the AggregatedInputEntry class and anywhere else there remains a reference to aggregatedInput

Reply:
this function was used to fetch data for the csv export only. Since we discussed to remove the "export csv" functionality it is no longer needed. I already deleted it in the refactoring branch.